### DISCUSSION TITLE
What does NWG Stand for?

### DISCUSSION THREAD

**Discussion by ek9** [nwg-piotr/nwg-shell#54] [Q&A] [answered]
Hi,

Great project. What does NWG stand for? I couldn't seem to find an answer to this.

**Comment by nwg-piotr** [comment, marked as answer]
Thank you!

LOL, actually nothing. In 2002 - 2015 I ran a local news website called "Nawigator" (en: Navigator). As the name was long, and the domain already taken, we used the nwg.pl domain. After that, I kept this domain for a couple of years. Today for me "nwg" is just a nickname that is usually not taken yet, and which I got used to.
